android/app/build.gradle

```diff
@@ -13,6 +13,7 @@ configurations.all {
     all*.exclude module: 'bcprov-jdk15on'
     all*.exclude module: 'bcutil-jdk15on'
     all*.exclude module: 'titanium-json-ld'
+    all*.exclude module: 'bcpkix-jdk15on'
```

Comment:
remove this as fix for this will be provided separately. https://github.com/mosip/inji-wallet/pull/1663

Reply:
Removed